### ISSUE TITLE
Bring your help to improve anything you want

### ISSUE THREAD

**Issue by pBouillon** [pBouillon/Novigrad#4]
This project is a mess on several parts:
- [x] awful packaging
- [x] approximate sqlite usage
- [x] ugly PDF
- [x] strange combination of sqlite/csv
etc.

Feel free to contribute !

**Comment by vishnuramana** [comment]
@pBouillon  Can I work on this ? I can help with the packages and PDF. But need some time to see how it works and all.

**Comment by pBouillon** [comment]
@V1cHu of course, it's all open ! Thanks for your interest and don't forget to put your name in credentials at the end too !
